server/auth/types/authentication_type.ts

```diff
@@ -277,6 +277,10 @@ export abstract class AuthenticationType implements IAuthenticationType {
     request: OpenSearchDashboardsRequest,
     authInfo: any
   ): SecuritySessionCookie;
+  public abstract getKeepAliveExpiry(
```

Comment:
Since many auth types use the same implementation of `now + session.ttl`, can this define a default implementation and then it doesn't need to be explicitly defined in the auth types that use the default implementation?

Reply:
I'm wondering if there is a better name for this? Maybe `extendSession`? wdyt?

Reply:
I think the method is named appropriately since it is hidden behind the flag checking whether keepalive is enabled or not, and the method is simply calculating the new expiry of the cookie after keep alive. I added a default implementation as well.